File: src/main/java/net/kigawa/kutil/unit/exception/NoSingleUnitException.java
package net.kigawa.kutil.unit.exception;

public class NoSingleUnitException extends RuntimeUnitException
{
    public NoSingleUnitException(String message, Throwable cause)
    {
        super(message, cause);
    }

    public NoSingleUnitException(String message)
    {
        super(message);
    }
}

File: src/main/java/net/kigawa/kutil/unit/exception/NoSingleUnitException.kt
package net.kigawa.kutil.unit.exception

@Suppress("unused")
class NoSingleUnitException: UnitException {
  constructor(message: String, cause: Throwable?, vararg obj: Any?): super(message, cause, *obj)
  constructor(message: String, vararg obj: Any?): super(message, *obj)
}